pcalcl.l

```diff
@@ -81,7 +81,30 @@ extern  int lineno;
                                 return(NUMBER);
                                 }
 
-[0-9]+([0-9_,]*[0-9]+)?         {
+KB                              {
+                                debug(" Memory size: [ %s ]\n", yytext);
+                                yylval.val = 1000;
+                                return(MEMORY_SIZE);
+                                }
+
+[KkMmGgTtPp] |
+[KMGTPE](iB)                   {
```

Comment:
this would be `[KkMmGgTtPpEeZzYy](iB)?`

Reply:
i understand, but i've been working on that, so might as well include ZY now and be done